signup.php

```diff
@@ -48,21 +41,23 @@
 					$Password = $_POST['password'];
 
 				//create a variable to store sql code for the ''And Users' query
-				$insertquery = "INSERT INTO User(User_ID, Password) VALUES ('".$User_ID."', '".$Password."')";
+				$insertquery = "INSERT INTO User(User, Password) VALUES ('".$User_ID."', '".$Password."')";
```

Comment:
I have change the name for the user column of the user table to User so it I need to change it in this code too so it can be function correctly.